tools/js-sdk-release-tools/src/hlcCodeGenCli.ts

```diff
@@ -9,7 +9,7 @@ const shell = require('shelljs');
 async function automationGenerateInTerminal(absoluteReadmeMd: string, tag?: string, use?: string, additionalArgs?: string) {
     const regexResult = /^(.*[\/\\]azure-rest-api-specs[-pr]*)[\/\\](specification.*)/.exec(absoluteReadmeMd);
     if (!regexResult || regexResult.length !== 3) {
-        logger.error(`Cannot Parse readme file path: ${absoluteReadmeMd}`);
+        logger.error(`Failed to parse README file path: '${absoluteReadmeMd}'.`);
```

Comment:
Can you add next steps to the message? such as "Ensure the READMD file: ${absoluteReadmeMd} exist and is valid. Refer to the sample file at https://github.com/Azure/azure-rest-api-specs/tree/main/documentation/samplefiles .